server/backend/cron/movies.go

```diff
@@ -0,0 +1,236 @@
+package cron
+
+import (
+	"database/sql"
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	log "github.com/sirupsen/logrus"
+)
+
+type MovieItems struct {
+	Title     string `xml:"title"`
+	Link      string `xml:"link"`
+	PubDate   string `xml:"pubDate"`
+	Location  string `xml:"location"`
+	Enclosure struct {
+		Url    string `xml:"url,attr"`
+		Length string `xml:"length,attr"`
+		Type   string `xml:"type,attr"`
+	} `xml:"enclosure"`
+}
+
+type MovieChannel struct {
+	Items []MovieItems `xml:"item"`
+}
+
+const (
+	MovieImageDirectory = "movie/"
+)
+
+func (c *CronService) movieCron(cronjob *model.Crontab) error {
+	//Get the news feed we want to get our data from
```

Comment:
Or better yet: just move the URL from the news sources to here. Its not a real news source anyway and its a bit confusing to read "news" inside of the movie processing.